src/components/sharedComponents/TokenInput/index.tsx

```diff
@@ -293,7 +293,7 @@ function TokenAmountField({
       onValueChange={({ value }) => onChange?.(value)}
       thousandSeparator={thousandSeparator}
       // NumericFormat has defaultValue prop overwritten and is not compatible with the standard
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      // biome-ignore lint/suspicious/noExplicitAny: <explanation>
```

Comment:
maybe the explanation that is in line 295 must go in the `<explanation>` placeholder?